main.js

```diff
@@ -14,6 +14,82 @@ const buttonBookmarks = getEl('#footer__bookmarks')
 const buttonCreate = getEl('#footer__create')
 const buttonProfile = getEl('#footer__profile')
 
+// Card data
+
+const cardData = [
+  {
+    question: 'What nut is used to make marzipan?',
+    answer: 'Almonds',
+    isBookmarked: true,
+    showAnswer: false,
+    tags: ['nature', 'food', 'cooking'],
+  },
+  {
+    question: 'How many letters in the word hippopotamus?',
+    answer: '12',
+    isBookmarked: true,
+    showAnswer: false,
+    tags: ['nature', 'animals', 'count'],
+  },
+  {
+    question: 'Who did Orlando Bloom play in Pirates Of The Caribbean?',
+    answer: 'Will Turner',
+    isBookmarked: true,
+    showAnswer: false,
+    tags: ['people', 'movies', 'names'],
+  },
+  {
+    question:
+      'What is the answer to the ultimate question of life, the universe, and everything?',
+    answer: '42',
+    isBookmarked: true,
+    showAnswer: false,
+    tags: ['fun', 'books', 'pop culture'],
+  },
+]
+
+function renderCard(cardData) {
+  const cardSection = document.createElement('section')
+  cardSection.classList.add('card')
+  mainHome.appendChild(cardSection)
+
+  const cardBookmark = document.createElement('button')
+  cardBookmark.classList.add('card__bookmark')
+  cardBookmark.setAttribute('aria-label', 'Bookmark')
+  cardSection.appendChild(cardBookmark)
+
+  const cardQuestion = document.createElement('h2')
+  cardQuestion.classList.add('card__question')
+  cardQuestion.textContent = cardData.question
+  cardSection.appendChild(cardQuestion)
+
+  const cardButton = document.createElement('button')
+  cardButton.classList.add('card__button')
+  cardButton.textContent = 'Show answer'
+  cardSection.appendChild(cardButton)
+
+  const cardAnswer = document.createElement('p')
+  cardAnswer.classList.add('card__answer')
+  cardAnswer.classList.add('card__answer--hidden')
```

Comment:
Since you have it in your `cardData` as well, I think an `if` statement would be good here:
```javascript
if(!cardData.showAnswer) {
    cardAnswer.classList.add('card__answer--hidden')
}
```
Which means that if `showAnser` is false, set the class to `card__answer--hidden`.